public/content/en/welcome.md

```diff
@@ -7,6 +7,31 @@ This tour will give you an overview of this powerful and expressive systems
 programming language which compiles directly to efficient, *native*
 machine code.
 
+### What is D
+
+D has many [unique features](http://dlang.org/overview.html) and is a
+general purpose systems and applications programming language.
+It is a _high level language_, but retains the ability to write _high
+performance_ code and interface directly with the _operating system API's_
+and with hardware.
+D is well suited to writing medium to large-scale programs with many million
+lines of code.
```

Comment:
I would just drop the "with .." sentence and replace it with "D is well suited to writing medium to large-scale programs". I think the reader will know what we mean by that.